pokedex/db/load.py

```diff
@@ -193,7 +193,35 @@ def load(session, tables=[], directory=None, drop_tables=False, verbose=False, s
 
     print_start('Creating tables')
     for n, table in enumerate(table_objs):
-        table.create(bind=engine)
+        try:
+            table.create(bind=engine)
+        
+        # Exceptions for handling the error thrown when trying to load
+        # the database with a table that already exists.
+        except (
+            sqlalchemy.exc.OperationalError,  # Exception used for SQLite
+            sqlalchemy.exc.ProgrammingError,  # Exception used for PostgreSQL
+            sqlalchemy.exc.InternalError      # Exception used for MySQL
+            ) as error:
+
+            if "already exists" in str(error.orig):
+                print("\n\nERROR:  The table '{}' already exists in the database. "
+                    "Did you mean to use 'pokedex load -D'".format(table))
+                sys.exit(1)
+
+            # If it happens to be some other error but raised by the same 
+            # exception, then an unexpected error message is sent with 
+            # the error included
+            else:
+                print("\n\n UNEXPECTED ERROR: ", error)
+                sys.exit(1)
+        
+        # In case any other exception not specified above is raised, an
+        # unexpected exception error is sent with the error included.
+        except Exception as error:
```

Comment:
Let's drop this except block. If the error isn't an expected error from the database or sqlalchemy, then it could be anything. Maybe we tickled a bug deep in sqlalchemy. Maybe it's an AttributeError because `table` is None. I think i'd rather see the traceback in that case.

(In general, catch-all except blocks should be used sparingly.)

Reply:
Good point I'll remove it.